Fabric.Identity.API/scripts/tests/Install-Identity-Utilities.tests.ps1

```diff
@@ -782,80 +782,35 @@ Describe 'Get-ClientSettingsFromInstallConfig' -Tag 'Unit' {
             $secondApp.clientSecret | Should -Be "secret2"
         }
     }
-}
-
-Describe 'Get-WebDeployPackagePath'{
```

Comment:
They are broken, and I don't think it is that important to test what zip file the file is in. But maybe, since the scripts are using that location to pull from.

Reply:
I added them back